pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -49,6 +49,9 @@ var (
 func NewCmdRolloutHistory(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &HistoryOptions{}
 
+	validArgs := []string{"deployment"}
```

Comment:
@kargakis, any suggestions to make it configurable? or @fabianofranz can take care of this in a new PR?